### PR TITLE
Add multiple po files support

### PR DESCRIPTION
**Pull request by wytrych**
Add possibility to use the gettext checker with a list of files or with wildcards with `grunt.file.expand`

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js Please review

**Comment by henrahmagix** [comment]
@wytrych See [`this.files`](http://gruntjs.com/api/inside-tasks#this.files) in the Grunt api: you can use that instead of using an option. At the same time, the default options shouldn't specify a default po file

File: tasks/gettext-checker.js [hunk 1]
```diff
@@ -32,87 +32,104 @@ module.exports = function (grunt) {
...
             if (!poObject.headers) {
                 grunt.fail.fatal(filePath + ' can not be loaded. Check the file is a valid .po file');
             }
         };
 
-        validatePoFile(templateFile, options.templateFile);
-        validatePoFile(poFile, options.poFile);
+        var error = false;
 
-        var getPoKeys = function (items) {
```
Comment (henrahmagix) [inline tasks/gettext-checker.js]: Why move this function?

**Comment by wytrych** [comment]
@henrahmagix Updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
-## 2.0.0
+# upcoming
+
+* Allow to use mutliple `.po` files
+* BREAKING CHANGE: files are now longer defined in `options`, use standard grunt `files` object:
```
Comment (henrahmagix) [inline CHANGELOG.md]: `now longer` -> `no longer`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,19 @@
...
+```js
+'gettext-checker': {
+    all: {
+        files: {
+            'i18n/template.pot': 'i18n/*.po'
```
Comment (henrahmagix) [inline CHANGELOG.md]: Can you explain here or in the readme that the DEST should be the catalog and the SRC should be the individual langauges?

File: tasks/gettext-checker.js [hunk 4]
```diff
@@ -32,87 +30,105 @@ module.exports = function (grunt) {
...
+        // get pot file keys as array
+        var potKeys = getPoKeys(templateFile.items);
+
+        var poFiles = this.filesSrc;
+        poFiles.forEach(function (poFile) {
+            checkPoFile(poFile, templateFile, templateFilePath, potKeys);
+        });
+
+        if (error) {
+            grunt.fail.fatal('Check FAILED.');
```
Comment (henrahmagix) [inline tasks/gettext-checker.js]: This is a change, only failing at the end of all tests: please update changelog

**Comment by wytrych** [comment]
Updated

**Pull request merged**